include/flutter-pi.h

```diff
@@ -313,7 +313,7 @@ struct frame {
 struct compositor;
 
 enum flutter_runtime_mode {
-	kDebug, kRelease
+	kDebug, kProfile, kRelease
```

Comment:
I just noticed, you added `kProfile` as a runtime mode and `--profile` to the usage, but you never added anything to the commandline args parsing that'll recognize that argument and also you didn't add anything to the flutter engine loading that'll load `libflutter_engine.so.profile` if profile mode was chosen.

Reply:
where do I find the flutter engine loading , as well as commandline args parsing ?

Reply:
flutter engine loading is here:
https://github.com/ardera/flutter-pi/blob/e306d1b9ce8766c82f2415b1f4f7ae6ae91f90e3/src/flutter-pi.c#L1816

cmdline arg parsing is here:
https://github.com/ardera/flutter-pi/blob/e306d1b9ce8766c82f2415b1f4f7ae6ae91f90e3/src/flutter-pi.c#L2300

basically, just search for `kRelease` or `kDebug` in flutter-pi.c and ensure for every occurence that you handle `kProfile` there correctly